issuance/cert.go

```diff
@@ -60,12 +62,20 @@ type Profile struct {
 
 // NewProfile synthesizes the profile config and issuer config into a single
 // object, and checks various aspects for correctness.
-func NewProfile(profileConfig ProfileConfig, skipLints []string) (*Profile, error) {
+func NewProfile(profileConfig ProfileConfig, skipLints []string) (*Profile, [32]byte, error) {
 	reg, err := linter.NewRegistry(skipLints)
 	if err != nil {
-		return nil, fmt.Errorf("creating lint registry: %w", err)
+		return nil, [32]byte{}, fmt.Errorf("creating lint registry: %w", err)
 	}
 
+	var encodedProfile bytes.Buffer
+	enc := gob.NewEncoder(&encodedProfile)
+	err = enc.Encode(profileConfig)
```

Comment:
```suggestion
	err = gob.NewEncoder(&encodedProfile).Encode(profileConfig)
```

But more importantly: this code should be in //cmd/boulder-ca/main.go, or in ca.go's makeCertificateProfilesMap -- just like the issuance package doesn't know about Names, it doesn't need to know about Hashes either.